common/display/src/main/scala/weco/catalogue/display_model/models/GetIdentifiers.scala

```diff
@@ -3,10 +3,15 @@ package weco.catalogue.display_model.models
 import weco.catalogue.internal_model.identifiers.IdState
 
 trait GetIdentifiers {
-  protected def getIdentifiers(id: IdState, includesIdentifiers: Boolean) =
+  protected def getIdentifiers(id: IdState): List[DisplayIdentifier] =
+    id.allSourceIdentifiers.map(DisplayIdentifier(_))
+
+  protected def getIdentifiers(
+    id: IdState,
+    includesIdentifiers: Boolean
+  ): Option[List[DisplayIdentifier]] =
```

Comment:
Given the changes in DisplayWork, which I assume mean that if those list values were absent, they are now empty lists, would it be appropriate for this to behave in the same way?

i.e.  not `Option[List...] ... else None`, but always return a lis, which will be empty if includesIdentifiers is false.

Is there a meaningful difference elsewhere between the identifiers option/list being None vs being empty?

(even if this would be better as an empty list, I can see that it would be a bit too widespread a change to include in this pull)

Reply:
Unfortunately not, or at least not without changing the outward behaviour of the API.

If you fetch images `/images?include=contributors`, it includes the contributors from the source work. Contributors can be identified, but there's no way to tell the images endpoint "please include/exclude identifiers in the response", so we omit identifiers on images.

When we construct a DisplayContributor from an internal model Contributor, we pass `includeIdentifiers = true|false` depending on whether we're in a Work or an Image context.